bsoid_app.py

```diff
@@ -1,3 +1,18 @@
+import os
+
+working_dir = os.environ.get('WORKING_DIR_BSOID', '')
```

Comment:
I'd move these to after the imports

Reply:
Given these are constants, I'd capitalize the names as well (like `WORKING_DIR`). Sorry for the annoying change!